tools/sdk-generation-pipeline/packages/sdk-generation-lib/src/types/publishResultConfig.ts

```diff
@@ -0,0 +1,190 @@
+import * as convict from 'convict';
+
+import { RepoType, SDK } from './commonType';
+
+export interface RepoInfo {
+    type: string;
+    path: string;
+    branch: string;
+}
+
+export class PublishResultConfig {
```

Comment:
Do we need this config which combine mogodb with repos?

Reply:
these parameters are encrypted field, don't want to show it in command line even it's ‘***’ in pipeline logs